config/virtuoso/virtuoso.ini

```diff
@@ -0,0 +1,262 @@
+ServerRoot			= /usr/local/virtuoso-opensource/var/lib/virtuoso/vsp
+MaxClientConnections		= 10
+DavRoot				= DAV
+EnabledDavVSP			= 0
+HTTPProxyEnabled		= 0
+TempASPXDir			= 0
+DefaultMailServer		= localhost:25
+ServerThreads			= 10
+MaxKeepAlives			= 10
+KeepAliveTimeout		= 10
+MaxCachedProxyConnections	= 10
+ProxyConnectionCacheTimeout	= 15
+HTTPThreadSize			= 280000
+HttpPrintWarningsInOutput	= 0
+Charset				= UTF-8
+;;HTTPLogFile		        = http.log
+MaintenancePage             	= atomic.html
+EnabledGzipContent          	= 1
+
+
+[AutoRepair]
+BadParentLinks			= 0
+
+[Client]
+SQL_PREFETCH_ROWS		= 100
+SQL_PREFETCH_BYTES		= 16000
+SQL_QUERY_TIMEOUT		= 0
+SQL_TXN_TIMEOUT			= 0
+;SQL_NO_CHAR_C_ESCAPE		= 1
+;SQL_UTF8_EXECS			= 0
+;SQL_NO_SYSTEM_TABLES		= 0
+;SQL_BINARY_TIMESTAMP		= 1
+;SQL_ENCRYPTION_ON_PASSWORD	= -1
+
+[VDB]
+ArrayOptimization		= 0
+NumArrayParameters		= 10
+VDBDisconnectTimeout		= 1000
+KeepConnectionOnFixedThread	= 0
+
+[Replication]
+ServerName			= db-D602566B774E
+ServerEnable			= 1
+QueueMax			= 50000
+
+
+;
+;  Striping setup
+;
+;  These parameters have only effect when Striping is set to 1 in the
+;  [Database] section, in which case the DatabaseFile parameter is ignored.
+;
+;  With striping, the database is spawned across multiple segments
+;  where each segment can have multiple stripes.
+;
+;  Format of the lines below:
+;    Segment<number> = <size>, <stripe file name> [, <stripe file name> .. ]
+;
+;  <number> must be ordered from 1 up.
+;
+;  The <size> is the total size of the segment which is equally divided
+;  across all stripes forming  the segment. Its specification can be in
+;  gigabytes (g), megabytes (m), kilobytes (k) or in database blocks
+;  (b, the default)
+;
+;  Note that the segment size must be a multiple of the database page size
+;  which is currently 8k. Also, the segment size must be divisible by the
+;  number of stripe files forming  the segment.
+;
+;  The example below creates a 200 meg database striped on two segments
+;  with two stripes of 50 meg and one of 100 meg.
+;
+;  You can always add more segments to the configuration, but once
+;  added, do not change the setup.
+;
+[Striping]
+Segment1			= 100M, db-seg1-1.db, db-seg1-2.db
+Segment2			= 100M, db-seg2-1.db
+;...
+
+;[TempStriping]
+;Segment1			= 100M, db-seg1-1.db, db-seg1-2.db
+;Segment2			= 100M, db-seg2-1.db
+;...
+
+;[Ucms]
+;UcmPath			= <path>
+;Ucm1				= <file>
+;Ucm2				= <file>
+;...
+
+
+[Zero Config]
+ServerName			= virtuoso (D602566B774E)
+;ServerDSN			= ZDSN
+;SSLServerName			= 
+;SSLServerDSN			= 
+
+
+[Mono]
+;MONO_TRACE			= Off
+;MONO_PATH			= <path_here>
+;MONO_ROOT			= <path_here>
+;MONO_CFG_DIR			= <path_here>
+;virtclr.dll			=
+
+
+[URIQA]
+DynamicLocal			= 0
+DefaultHost			= localhost:8890
+
+
+[SPARQL]
+;ExternalQuerySource		= 1
+;ExternalXsltSource 		= 1
+;DefaultGraph      		= http://localhost:8890/dataspace
+;ImmutableGraphs    		= http://localhost:8890/dataspace
+ResultSetMaxRows           	= 10000
```

Comment:
lets put this one higher too, same for maxconstruct triples. I would remove the max query cost estimation time and set max to something a lot higher like 600?